Sources/App/WebView/WebViewController.swift

```diff
@@ -136,6 +136,7 @@ final class WebViewController: UIViewController, WKNavigationDelegate, WKUIDeleg
 
         config.userContentController = userContentController
         config.applicationNameForUserAgent = HomeAssistantAPI.applicationNameForUserAgent
+        config.defaultWebpagePreferences.preferredContentMode = Current.isCatalyst ? .desktop : .mobile
```

Comment:
If desktop is still used for Mac, I think the resulting user agent string would still unfortunately result in always serving the legacy frontend build, correct?

Reply:
If I set "mobile" to be used for Mac the result user agent is even worse:

`Mozilla/5.0 (iPad; CPU OS 14_5 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Home Assistant/2024.8 (io.robbie.HomeAssistant.dev; build:2024; macOS 14.5.0) Mobile/HomeAssistant, like Safari`

Says "iPad" and "OS 14_5" which is not the iPad version (it's macOS version)

Reply:
Ugh, yeah that's terrible.  Like I said, Apple's fault.

Reply:
Indeed and looking at historic data, they are not working to improve this 😅